index.ts

```diff
@@ -214,6 +243,22 @@ async function disburseNeuron(neuronId: bigint, to?: string, amount?: bigint) {
   ok();
 }
 
+async function splitNeuron(neuronId: bigint, amount: bigint) {
+  const identity = await getLedgerIdentity();
```

Comment:
hint: this piece of code is repeated few times, can probably be refactored to avoid duplication

```
  const identity = await getLedgerIdentity();
  await assertLedgerVersion({ identity, minVersion: "2.1.7" });
  const governance = GovernanceCanister.create({
    agent: await getAgent(identity),
    hardwareWallet: false,
  });
```

Reply:
I assume the code duplication in favor of readability and clarity.

Reply:
That's the thing, avoiding these duplication would have ease reading this PR in the review pane of GitHub at least.